### PR TITLE
Connect with us section 

### LINKED ISSUE #174: Create a connect with us section containing the links provided in the footer
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [index.html](https://github.com/publiclab/community-toolbox/blob/main/index.html) and [demo.css](https://github.com/publiclab/community-toolbox/blob/main/examples/demo.css) in the `community-toolbox` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

_In **index.html**_

- On **line 144** add these changes,

```diff
$DIFF
+ <div class="row" id="connectwithus">
+ <h2 class="text-center d-block">Connect with us</h2>
+ <p>You can connect with us on any of these platforms:</pc>
+ <div class="text-center" id="connectwithusbody">
+ <a class="col-md-2" href="https://www.facebook.com/PublicLab/" title="Facebook" ><i class="fa fa-facebook fa-3x"></i> </a>
+ <a class="col-md-2" href="https://twitter.com/PublicLab?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor" title="Twitter" ><i class="fa fa-twitter fa-3x"></i> </a>
+ <a class="col-md-2" href="https://github.com/publiclab" title="Organization" ><i class="fa fa-github fa-3x"></i> </a>
+ <a class="col-md-2" href="https://publiclab.org" title="Publiclab Website"><i class="fa fa-globe fa-3x"></i></a>
+ </div>
+ </div>
```

- From **line 155** upto **line 161**, remove everything,

```diff
$DIFF
- <div class="col-md-6 text-center">
- <span class="float-md-right">
- <a href="https://www.facebook.com/PublicLab/" title="Facebook" ><i class="fa fa-facebook"></i> </a>
- <a href="https://twitter.com/PublicLab?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor" title="Twitter" ><i class="fa fa-twitter"></i> </a>
- <a href="https://github.com/publiclab" title="Organization" ><i class="fa fa-github"></i> </a>
- <a href="https://publiclab.org" title="Publiclab Website"><i class="fa fa-globe"></i> Publiclab</a> </span>
- </div>
```

_On **demo.css**_

```diff
$DIFF
+ #connectwithus {
+ margin: auto auto 5em;
+ }
+ #connectwithusbody {
+ overflow: hidden;
+ }
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by ibrahimm02**
Fixes #174

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by gauravano** [comment]
Hi @ibrahimm02, 

You were supposed to add issue number in the PR comment - https://github.com/publiclab/community-toolbox/pull/178#issue-270260250. I added for you this time.

Also, it's nice to create a separate feature branch for the PR so that `main` branch remain clean. 
Please mark the checkboxes too!

Thanks!

File: index.html [hunk 1]
```diff
@@ -141,7 +141,17 @@ <h2 class="contributors-head">Contributors</h2>
       <p>This project was made possible by many contributors, including the following GitHub users:</p>
 
       <div class="well contributors"><div id="spinner-icon" style="text-align: center;"><i class="fa fa-spinner fa-spin"></i></div><p class="usernames"></p><hr /><p class="avatars"></p></div>
-
+	
+	  <div class="row" id="connectwithus">
+		<h2 class="text-center d-block">Connect with us</h2>
+		<p>You can connect with us on any of these platforms:</pc>
```
Comment (gauravano) [inline index.html]:
```suggestion
		<p>You can connect with us on any of these platforms:</p>
```

Comment (subhahu123) [reply index.html]: @gauravano  we should html validation to grunt which will automatically detect this type of error

**Comment by ibrahimm02** [comment]
Hi @gauravano, thanks for your review. I will make changes as suggested.

File: index.html (hunk 1 above)
Comment (gauravano) [reply index.html]: Yes, we can surely do this! Nice idea

**Comment by gauravano** [comment]
Okay, thanks @ibrahimm02. Also, there's a merge conflict so please resolve that too. Thanks!

**Comment by Rishabh570** [comment]
Closing this PR, the latest version of it is merged!

**Pull request closed without merging**